events.go

```diff
@@ -74,8 +74,14 @@ type cpuUsage struct {
 	User   uint64   `json:"user"`
 }
 
+type cfs struct {
+	Period uint64 `json:"period,omitempty"`
+	Quota  int64  `json:"quota,omitempty"`
+}
+
 type cpu struct {
 	Usage      cpuUsage   `json:"usage,omitempty"`
+	Scheduling cfs        `json:"cfs,omitempty"`
```

Comment:
actually, CPU.CpuCfs is redunant, that one should be CFS.  So the fields will be called Cfs and the json fields, in order to match the others, will be cpu_cfs